llama_hub/tools/shopify/base.py

```diff
@@ -0,0 +1,36 @@
+"""Shopify tool spec."""
+
+from llama_index.tools.tool_spec.base import BaseToolSpec
+from typing import Optional
+import shopify
+
+class ShopifyToolSpec(BaseToolSpec):
```

Comment:
out of curiosity could we create a general "GraphQLAPIToolSpec" that could run graphql queries against any graphql http endpoint? 

to be clear we should def keep this one

Reply:
Will work on splitting out a GraphQLAPIToolSpec and SchemaDefinitionLanguageLoader today 👌